.github/workflows/test.yml

```diff
@@ -125,19 +125,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.10', '3.11']
-        django-version: ['3.2', '4.2', '5.0']
+        python-version: ['3.11', '3.12']
+        django-version: ['4.2', '5.0']
         include:
-          - python-version: '3.8'
-            django-version: '3.2'
           - python-version: '3.8'
             django-version: '4.2'
           - python-version: '3.9'
-            django-version: '3.2'
-          - python-version: '3.9'
             django-version: '4.2'
-          - python-version: '3.12'
-            django-version: '4.1'
```

Comment:
Oh, I didn't realize Django 4.1 has been EOL for so long already.

We should remove Django 4.1 "partial support" from README as well.